### ISSUE TITLE
Cannot scan with Python3

### ISSUE THREAD

**Issue by dkarchmer** [iotile/coretools#459]
Packages:
```
pip install --upgrade iotile-core iotile-transport-bled112 iotile-support-firm-arduino-bridge-0 iotile-support-con-nrf52832-2 iotile-support-lib-controller-3
```

Command:

```
iotile hw --port=bled112 scan
```

On python2, **it works.**

On python3:

```
(poduino2) david$ iotile hw --port=bled112 scan
Traceback (most recent call last):
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/iotile/core/scripts/iotile_script.py", line 208, in main
    finished = shell.invoke(line)
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/typedargs/shell.py", line 494, in invoke
    val, line, finished = self.invoke_one(line)
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/typedargs/shell.py", line 457, in invoke_one
    val = func(*posargs, **kwargs)
  File "<decorator-gen-13>", line 2, in __init__
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/typedargs/utils.py", line 34, in _check_and_execute
    retval = func(*convargs, **convkw)
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/iotile/core/hw/hwmanager.py", line 77, in __init__
    self.stream = self._create_stream()
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/iotile/core/hw/hwmanager.py", line 854, in _create_stream
    adapter_factory = adapter_entry.load()
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/david/.virtualenv/poduino2/lib/python3.6/site-packages/iotile_transport_bled112/bled112.py", line 4, in <module>
    from Queue import Queue
ModuleNotFoundError: No module named 'Queue'
```

**Comment by timburke** [comment]
`iotile-transport-bled112` has not been ported to python 3 yet.  The current status of what has and has not been ported is here (one line with a True, False flag per package for whether it is currently known to be python 3 compatible):

https://github.com/iotile/coretools/blob/master/scripts/components.py#L1

**Comment by timburke** [comment]
This should be resolved in `iotile-transport-bled112 v1.7.1`
